### ISSUE TITLE
Commandtag Writefile adds a comma when making use of "

### ISSUE THREAD

**Issue by zpammer** [PhantomBot/PhantomBot#2907]
PhantomBot Version: 3.6.4.5-NB-08212022 (nightly_build) (Revision: 39db73f)
O/S Version: Windows Server 2022 21H2
Java Version: windows build in 11.0.14.1+1
Web Browser Version (if Panel related): 1.6.12

**Detailed Description: **
When in a writefile the ``"`` is being used it results into comma's being added before and/or after the ``"``

**Steps to Reproduce:**
Command: ``(writefile steamsign.txt, false, <h1 style="font-size:4vw; font-family:'Lucida Handwriting'"> Marte! </h1> 
Please sign the following profiles and clean the list with <B> !steamclear </B> <img src=emotes\marteiWine.png width=28 height=28> <BR><BR>) marteiNotes Okay, I cleaned the list!``

Logs: -

Screenshots (if applicable):
![MIEWjvJNYF](https://user-images.githubusercontent.com/11378079/187085548-d02c2761-81b4-4828-adc1-7854c594dafb.png)

**Comment by gmt2001** [comment]
The argument must be wrapped in quotes and literal quotes must be escaped

**Comment by gmt2001** [comment]
Try again without escaping or wrapping in quotes in tomorrow's nightly

**Comment by zpammer** [comment]
I tried without wrapping in quotes and that did not give the desired outcome to build up the HTML.

But I tried it with quotes exactly as in the supplied screenshot and it does not write the extra comma's anymore and that is the desired output!


Regarding https://github.com/PhantomBot/PhantomBot/issues/2899
I want to share the following update and I will post back onto that issue in full detail what I tried.

Requires some time as I try different things to pin the cause.
I had set several keepalive windows registry settings to keep a better connection to discord, but they are not required anymore due to the bot posting delayed when a connection has been lost/closed.
So I hoped to see no issues anymore, but...
So next try is to set back the idle timeout from 30 back to the default = 4.
After that my idea is to run a copy of the bot from my PC at home with only the API commands.
